### ISSUE TITLE
Flagging User avatars

### ISSUE THREAD

**Issue by aaronschachter** [DoSomething/aurora#24]
Need to ability to flag a user avatar, as it's public in Lets Do This: https://github.com/DoSomething/product/blob/master/LetsDoThis/Screens/Hub/Public%20Hub*.png

**Comment by mshmsh5000** [comment]
You mean, if a member adds a raunchy avatar?

**Comment by aaronschachter** [comment]
Yup :neckbeard:.  In Lets Do This, the chances are slim, but a user could join a signup group, causing their avatar to display for others within the group, and then change the pic to something unpleasant.

**Comment by angaither** [comment]
We need to add support for avatars in aurora first :wink:
